libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -171,12 +171,57 @@ export const HealthInsuranceForm: Form = buildForm({
       ],
     }),
     buildSection({
-      id: 'info',
+      id: 'FormerCountryofInsuranceSection',
       name: '',
       children: [
-        buildTextField({
-          id: 'infoInput',
-          name: 'text input',
+        buildMultiField({
+          id: 'FormerCountryofInsurance',
+          name: m.formerCountryOfInsuranceTitle,
+          children: [
+            buildRadioField({
+              id: 'formerCountryofInsuranceRegistration',
+              name: '',
+              description: m.formerCountryOfInsuranceDescription,
+              largeButtons: true,
+              options: [
+                { label: m.formerCountryOfInsuranceNo, value: NO },
+                { label: m.yesOptionLabel, value: YES },
+              ],
+            }),
+            buildIntroductionField({
+              id: 'infoFormerCountryInsurance',
+              name: '',
+              introduction: m.formerCountryOfInsuranceInfo,
+            }),
+            buildTextField({
+              id: 'applicant.country',
```

Comment:
Perhaps this should be formerCountry so it isn't confused with previous screen with contact information